README.md

```diff
@@ -28,14 +30,25 @@ Or, if you want to remove all data
 
 When you start the container for the first time, a superuser will be created. Make sure you change the password for this user (the easiest way is through the Django shell).
```

Comment:
I think it would be nice to include a link to or short explanation for how to do this.